macos/FluentUI/Button/Button.swift

```diff
@@ -524,6 +524,7 @@ class ButtonCell: NSButtonCell {
 @objc(MSFButtonSize)
```

Comment:
Based on the current ButtonSizeParams logic, the Padding portions that account for height and width of the button are not absolute in that the dimensions of the button are auto-scaled to accommodate the size of the Image with it. Additionally documenting a height and width would only be duplicating the numbers already noted in the static variable assignments in ButtonSizeParamaters - so it would require a 2-place maintenance. So I'm questioning if it might be worth it as its straightforward to simply look closer at the sizes to gauge the dimensions?

Reply:
we document all of our public variables, enums, function. think of all Apple appkit documentation is standard.

Reply:
Maybe just add a quick comment for each size indicating the "minimum height is X, but the button will grow in height if needed to fit its image".

Reply:
Done. I've documented the button heights because they now have a minimum constant.

Reply:
Seems fine to me as-is, but perhaps it would be more accurate to say "28 pts minimum" or would that cause confusion?